groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -4317,13 +4365,17 @@ template <class TYPE>
 BSLS_KEYWORD_CONSTEXPR bool operator<(
                              const bsl::optional<TYPE>&,
                              const bsl::nullopt_t&) BSLS_KEYWORD_NOEXCEPT;
-    // Return 'false'.
+    // Return 'false'.  Note that 'bsl::nullopt_t' never orders before a
+    // 'bsl::optional'.
+
 
 template <class TYPE>
 BSLS_KEYWORD_CONSTEXPR bool operator<(
                        const bsl::nullopt_t&,
                        const bsl::optional<TYPE>& value) BSLS_KEYWORD_NOEXCEPT;
     // Return 'true' if specified 'value' is engaged, and 'false' otherwise.
+    // Note that 'bsl::nullopt_t' sorts before any 'bsl::optional' that is
```

Comment:
```suggestion
    // Note that 'bsl::nullopt_t' is ordered before any 'bsl::optional' that is
```

Reply:
done. Just like above, this is a direct copy paste from nullableValue. What's the rule on changing this one in NullableValue ?